### DISCUSSION TITLE
SkyWalking shows no Endpoint information

### DISCUSSION THREAD

**Discussion by h-mx** [apache/skywalking#9369] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

Skywlking has no Endpoint information, it can monitor kafka's endpoint information, it can monitor JVM information. There is no error log
JDK 17
Skywlking 9.2.0-SNAPSHOT
agent 8.11.0

### What you expected to happen

SkyWalking does not support non-Spring projects, I use the company's own framework.
![image](https://user-images.githubusercontent.com/71714188/180182738-abbcd4d7-bf1b-4fcb-9efd-2345589a2db1.png)
![image](https://user-images.githubusercontent.com/71714188/180181440-228e2051-4990-49be-9101-fc1460d62ce5.png)

### How to reproduce

jdk17
非spring项目

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I am not sure why your private framework should be considered for an open-source project. 
Your codes, your responsibility.

**Reply by h-mx** [reply]
Based on netty development of a Web framework.

**Reply by wu-sheng** [reply]
You need to write a plugin for your own. We don't care about your private things.
Remember this, all private things would not be considered by a community.
SkyWalking project is a place people work together. We don't provide a customer service. You need to learn the docs, and follow codes, before simply throw everything to the community.
